### ISSUE TITLE
displayAplicationUnique() or displayApplication() ?

### ISSUE THREAD

**Issue by Tich1212** [HydrologicEngineeringCenter/SimpleRTSPlugin#12]
@HenryGeorgist Should I be implementing my plugins actions `displayApplicationUnique()` or `displayApplication()`?

https://github.com/HydrologicEngineeringCenter/SimpleRTSPlugin/blob/71823a10efe1114da9d1fe584c3bd38ae1511561/src/simplertsplugin/SimpleRTSPluginMain.java#L91-L98

And why?

**Comment by HenryGeorgist** [comment]
@Tich1212 the name of the method is really irrelevant. It can be whatever you would like it to be. Just be sure the method name provided in display action is the same as the method name you implement and intend to fire with the click event.

**Comment by Tich1212** [comment]
`displayApplication()` is a method that is extended from SimpleRtsPlugin. In the WAT example, displayApplicationUnique() is one that we create ourselves and call our plugin action in `DisplayApplication()`;

https://github.com/HydrologicEngineeringCenter/SimpleRTSPlugin/blob/71823a10efe1114da9d1fe584c3bd38ae1511561/src/simplertsplugin/SimpleRTSPluginMain.java#L51-L52

Is there a reason why we don't just write our plugins actions in `displayApplicationUnique()`

**Comment by HenryGeorgist** [comment]
I created displayactionunique because I encountered a bug in wat. I found that the method "displayAction" was registered by another plugin and would fire instead of my method. It could have been a bug in my code, but I solved it by making a uniquely named method (by adding the word unique).

change the declaration on line 52 to point at "displayAction" and delete dis;lay action unique, I think that is the best/cleanest solution
